app/components/repo/RepoWorkItemList.vue

```
@@ -0,0 +1,317 @@
+<script setup lang="ts">
+import type { WorkItem } from '~~/shared/types/work-item'
+
+const props = defineProps<{
+  owner: string
+  repo: string
+  limit?: number
```

Biome (2.4.4)

[error] Duplicate key limit found in Vue component.

(lint/correctness/noVueDuplicateKeys)